### ISSUE TITLE
Could not find mathglConfig.cmake

### ISSUE THREAD

**Issue by linyuanchang** [microsoft/vcpkg#28285]
I got the following errors when trying to use the package mathgl.
```
Selecting Windows SDK version 10.0.22000.0 to target Windows 10.0.19044.
CMake Error at D:/vcpkg/scripts/buildsystems/vcpkg.cmake:843 (_find_package):
  Could not find a package configuration file provided by "mathgl" with any
  of the following names:

    mathglConfig.cmake
    mathgl-config.cmake

  Add the installation prefix of "mathgl" to CMAKE_PREFIX_PATH or set
  "mathgl_DIR" to a directory containing one of the above files.  If "mathgl"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  CMakeLists.txt:16 (find_package)
```

But I really find the follow files:
```
MathGL2Config.cmake
MathGL2Targets.cmake
```

Is this a bug relate to the version?

**Comment by dg0yt** [comment]
`MathGL2Config.cmake` create a case-sensitive package name `MathGL2`, and `MathGL` should work, too.

(`mathgl`  would  work with `mathgl2-config.cmake`.)

**Comment by dg0yt** [comment]
Duplicates #22264.

**Comment by LilyWangLL** [comment]
Fixed by #23811.
